.gitignore

```diff
@@ -22,6 +22,9 @@ src/
 # Comma-Separated Values (CSV) Reports
 *.csv
 
+#XLSX Reports
+*.xlsx
```

Comment:
add this inorder not to add the results that are exported with --xlsx parameter to github